substrate-node/pallets/pallet-smart-contract/src/tests.rs

```diff
@@ -1019,41 +1019,10 @@ fn test_node_contract_billing_details() {
 
         info!("total amount billed {:?}", total_amount_billed);
 
-        let staking_pool_account_balance = Balances::free_balance(&get_staking_pool_account());
-        info!(
-            "staking pool account balance, {:?}",
-            staking_pool_account_balance
-        );
-
-        // 5% is sent to the staking pool account
-        assert_eq!(
-            staking_pool_account_balance,
-            (Perbill::from_percent(5) * total_amount_billed) + EXISTENTIAL_DEPOSIT
-        );
-
-        // 10% is sent to the foundation account
-        let pricing_policy = TfgridModule::pricing_policies(1).unwrap();
-        let foundation_account_balance = Balances::free_balance(&pricing_policy.foundation_account);
-        assert_eq!(
-            foundation_account_balance,
-            (Perbill::from_percent(10) * total_amount_billed) + EXISTENTIAL_DEPOSIT
-        );
-
-        // 50% is sent to the sales account
-        let sales_account_balance = Balances::free_balance(&pricing_policy.certified_sales_account);
-        assert_eq!(
-            sales_account_balance,
-            (Perbill::from_percent(50) * total_amount_billed) + EXISTENTIAL_DEPOSIT
-        );
+        let total_balance = Balances::total_balance(&twin.account_id);
+        let total_amount_billed = initial_twin_balance - total_balance;
```

Comment:
I guess this is already calculate just above

Reply:
see below